README.md

```diff
@@ -294,18 +293,18 @@ service studentData on studentServiceListener {
         }
     }
 
+    // Resource configuration for deleting a students detail's from the system.
     @http:ResourceConfig {
         methods: ["GET"],
         path: "/deleteStu/{studentId}"
     }
-
     // Delete Students resource for deleteing a student using id.
     resource function deleteStudent(http:Caller caller, http:Request request, int studentId) {
         studentData.requestCounts += 1;
         http:Response response = new;
         json status = {};
 
-        // Calling the deleteData function with id as the parameter and get a return json object.
+        // Calling deleteData function with id as parameter and get a return json object.
```

Comment:
Calling the deleteData function with the studentId as the parameter and getting a JSON object returned.